exchanges/coinbene/coinbene.go

```diff
@@ -0,0 +1,342 @@
+package coinbene
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/config"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/websocket/wshandler"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+// Coinbene is the overarching type across this package
+type Coinbene struct {
+	exchange.Base
+	WebsocketConn *wshandler.WebsocketConnection
+}
+
+const (
+	coinbeneAPIURL     = "http://openapi-exchange.coinbene.com/api/exchange/"
```

Comment:
Looks like their HTTPS endpoint is back online: https://openapi-exchange.coinbene.com/api/exchange/v2/market/tradePair/list